bundles/org.eclipse.core.resources/src/org/eclipse/core/internal/resources/ValidateProjectEncoding.java

```diff
@@ -95,6 +97,19 @@ static void updateMissingEncodingMarker(IProject project) {
 		}
 	}
 
+	/**
+	 * Returns the current severity for
+	 * {@link ResourcesPlugin#PREF_MISSING_ENCODING_MARKER_SEVERITY}.
+	 *
+	 * @return current severity. Default value is nothing else is specified.
```

Comment:
I don't understand the sentence here, re-wording it would be great.

Reply:
Thanks. There was a typo involved here. Fixed it.